### PR TITLE
feat(RichText): Pass Custom DesignTokens into RichText

### PR DESCRIPTION
**Pull request by rawi96**

### CODE REVIEW

File: src/components/RichTextEditor/utils/getTextStyles.ts [hunk 1]
```diff
@@ -1,14 +1,13 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
```
Comment (celine-s) [inline src/components/RichTextEditor/utils/getTextStyles.ts:1]: Maybe rename it to something like `textStyles` as getTextStyle function is removed

File: src/components/RichTextEditor/RichTextEditor.tsx [hunk 2]
```diff
@@ -48,6 +48,28 @@ export const RichTextEditor: FC<RichTextEditorProps> = ({
         onBlur: () => onBlur && onBlur(JSON.stringify(localValue.current)),
     };
 
+    if (!designTokens.heading1) {
+        designTokens.heading1 = defaultDesignTokens.heading1;
+    }
+    if (!designTokens.heading2) {
```
Comment (celine-s) [inline src/components/RichTextEditor/RichTextEditor.tsx:54]: else if?

**Review by celine-s — COMMENTED** [review]
I just added some small comments. Overall it looks really good and works smoothly👌 🥳

File: src/components/RichTextEditor/RichTextEditor.tsx (hunk 2 above)
Comment (rawi96) [reply src/components/RichTextEditor/RichTextEditor.tsx:54]: There could be more than one style missing. With the else if we would lose it.

**Review by celine-s — APPROVED** [review]
nice! 🚀

**Pull request merged**